pkg/util/mount/mount_windows.go

```diff
@@ -219,9 +219,9 @@ func (mounter *Mounter) GetFileType(pathname string) (FileType, error) {
 		return FileTypeBlockDev, nil
 	case syscall.S_IFCHR:
```

Comment:
What about all these other types?

Reply:
Current method could not work well for some file types.

@msau42 So I filed another PR #62154 to address this issue.